src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
-body {
+@import './variables';
+@import './body';
+@import './grade';
+@import './price';
+
+p {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__items {
+  object-fit: contain;
+  margin: 32px 19px 40px;
```

Comment:
It seems like the `.card__items` class is meant to style images. If that's the case, `object-fit: contain;` is a property for images, but it's not clear if `.card__items` is applied to an `img` element. Ensure that this property is used with image elements. If `.card__items` is not an image, consider renaming the class to reflect its purpose more accurately.